Example/Tests/GINICameraViewControllerTests.swift

```diff
@@ -22,7 +27,8 @@ final class CameraViewControllerTests: XCTestCase {
     
     func testTooltipWhenFileImportDisabled() {
         ToolTipView.shouldShowFileImportToolTip = true
-        GiniConfiguration.sharedConfiguration.fileImportSupportedTypes = .none
+        giniConfiguration.fileImportSupportedTypes = .none
+        vc = CameraViewController(giniConfiguration: giniConfiguration)
```

Comment:
It's a bit confusing that you have a property called `vc` that's already initialized with a view controller and you are reassigning it with a new one.

Secondly, `vc` is not a good name ;)

Reply:
I do this because the `giniConfiguration` can be used in the initialization phase. 
Yes, it can have a better name.